twoliter/embedded/build.Dockerfile

```diff
@@ -152,11 +153,19 @@ RUN --mount=target=/host \
       KIT_REPOS+=("--repofrompath=${kit},/host/build/kits/${kit}/${ARCH}" --enablerepo "${kit}") ; \
     done && \
     echo "${KIT_REPOS[@]}" && \
+    declare -a EXTERNAL_KIT_REPOS && \
+    for kit in ${EXTERNAL_KIT_DEPENDENCIES} ; do \
+       REPO_NAME="$(tr -s '/' '-' <<< "${kit}")" && \
```

Comment:
Is this transform needed? Seems like the variable can list kits in the canonical `vendor/kit` format.

Reply:
The transformation is for the repository name in dnf/yum, I wasn't sure / are allowed for repository names

Reply:
If that transformation is required, can it be done in the rust code that provides the list instead?